docs/incidents/2024-08-28-site-outage.md

```diff
@@ -0,0 +1,22 @@
+# Site Outage
+
+## What happened?
+
+- 2024-08-28
+  - 09:?? - Opportunistic maintenance to remove stale container images in production began.
+  - 09:30 - The frontend microservice that provides the user-facing website became unavailable (HTTP 5xx responses).
+  - 15:?? - Admin attempts to use the website and notices that it is unavailable.
+  - 15:?? - Admin logs into production, notices that the frontend service is failing due to a lack of container images.
+  - 15:?? - Admin rebuilds and redeploys the latest copy of the frontend microservice.
+  - 15:45 - Availability of the frontend microservice was restored.
+
+## What went wrong?
+
+- Essential current container images were removed from production.
```

Comment:
The frontend website is an entirely static web application - it has no dynamic functionality and none of the files contained in it change between deployments.  Perhaps we could use a simpler mechanism than a container to serve it?

Reply:
A good idea, but no, the current arrangement is fine - the consistency benefit from managing every single microservice using kubernetes deployments and makefiles that are broadly similar seems worthwhile.